schema/project-level/codelists/projectStatus.csv

```diff
@@ -1,7 +1,7 @@
-Code,Title,Description
-identification,Identification,Identification refers to the decision to develop a project within the budget and programme of a project owner (the public entity responsible for executing the budget).
-preparation,Preparation,"Preparation covers the feasibility study, environmental and social impact assessment, general scoping of the project, establishing the packaging and procurement strategy, preliminary statutory requirements on environmental and land impacts, and the resulting budget authorisation."
-construction,construction,"Implementation or package delivery covers the procurement and implementation of the planning, design and works according to the procurement strategy."
-completion,Completion,"Completion covers the handover of the assets and close-out activities with details of the final scope, cost, and delivery time."
-completed,Completed,"Close-out activities have been completed, and this project is no longer active. This project may be displayed as part of a list of archived projects."
-cancelled,Cancelled,This project has been cancelled prior to close-out activities. Cancellation may have taken place at any point after identification.
+Code,Title,Description
+identification,Identification,Identification refers to the decision to develop a project within the budget and programme of a project owner (the public entity responsible for executing the budget).
+preparation,Preparation,"Preparation covers the feasibility study, environmental and social impact assessment, general scoping of the project, establishing the packaging and procurement strategy, preliminary statutory requirements on environmental and land impacts, and the resulting budget authorisation."
+implementation,implementation,"Implementation covers the procurement and implementation of the group of services, such as design and supervision, or works to be delivered under the project, including works or services performed by the procuring entity. This differs from the definition of implementation in OCDS which covers only the post-award phase of the contracting process."

```

Comment:
> Implementation covers the procurement and implementation of the group of works or services (such as design and supervision) to be delivered under the project, including works or services performed by the procuring entity.

I found the use of commas made this hard to parse.

> This differs from the definition of 'implementation' in OCDS, which covers the implementation but not the procurement.

Rather than introduce a new concept ("post-award phase"), I think it's clearer to refer to the concepts already covered in the first part of the definition.

Reply:
Fixed in https://github.com/open-contracting/infrastructure/pull/99/commits/1378cf84d3910c8a01fb11e6e15d74c2517b282f